packages/editor/src/components/post-text-editor/index.js

```diff
@@ -80,7 +80,7 @@ export class PostTextEditor extends Component {
 					onBlur={ this.stopEditing }
 					className="editor-post-text-editor"
 					id={ `post-content-${ instanceId }` }
-					placeholder={ decodedPlaceholder || __( 'Write your story' ) }
+					placeholder={ decodedPlaceholder || __( 'Type text or press “/” to insert a block' ) }
```

Comment:
You can't press "/" or insert a block. This is the text editor, you can just type HTML :)

Reply:
Oh, oops, right. I think it too had the “Write your story” prompt and I just got confused. I’ll update it, my bad! 😅

Reply:
This will actually still show because we pass the same placeholder to the code editor. We probably shouldn't do that, but it's a different issue...